src/lib/OpenEXRCore/internal_coding.h

```diff
@@ -112,9 +99,6 @@ exr_result_t internal_decode_alloc_buffer (
 static inline float
 half_to_float (uint16_t hv)
 {
-#ifdef IMATH_HALF_SAFE_FOR_C
```

Comment:
i had these in case we (eventually) inherit neon / other simd stuff into imath, is there a reason to remove this?